website/static/js/file-browser.js

```
@@ -0,0 +1,284 @@
+            { label : 'All My Projects', url : 'http://localhost:8000/v2/users/me/nodes/?filter%5Bregistration%5D=false', type : 'collection'}
+        ]);
+        self.nameFilters = [
+            { label : 'Caner Uguz', id : '8q36f', type : 'filter'}
+        ];
+        self.tagFilters = [
+            { tag : 'something', type : 'filter'}
+        ];
+
+        self.updateFilesData = function(linkObject) {
+            linkObject.link = self.generateLinks(linkObject);
+            if (linkObject.link !== self.filesData()) {
+                self.filesData(linkObject.link);
+                self.isLoadedUrl = false; // check if in fact changed
+                self.updateBreadcrumbs(linkObject);
+            }
+        };
+
+        // INFORMATION PANEL
+        self.selected = m.prop([]);
+        self.updateSelected = function(selectedList){
+            self.selected(selectedList);
+            console.log(self.selected());
+        }.bind(self);
+
+
+        // COLLECTIONS PANEL
+        self.activeCollection = m.prop(1);
+        self.updateCollection = function(coll) {
+            self.activeCollection(coll.id);
+            var linkObject = {
+                type : 'collection',
+                data : coll,
+                label : coll.label,
+                id : coll.id
+            };
+            self.updateFilesData(linkObject);
+        };
+
+
+        // USER FILTER
+        self.activeUser = m.prop(1);
+        self.updateUserFilter = function(user) {
+            self.activeUser(user.id);
+            var linkObject = {
+                type : 'user',
+                data : user,
+                label : user.label,
+                id : user.id
+            };
+            self.updateFilesData(linkObject);
+        };
+
+        // Refresh the Grid
+        self.updateList = function(element, isInit, context){
+            if(!self.isLoadedUrl) {
+                var el = element || document.getElementById('pOrganizer');
```

Is `element` ever variable? Can it ever be undefined?

Falling back to a hard-coded id seems a bit dangerous (although hopefully we never have more than one PO on a page)